src/handlers/notebook.ts

```diff
@@ -23,9 +25,10 @@ export class NotebookHandler implements IDisposable {
     this.debuggerService = options.debuggerService;
     this.notebookPanel = options.widget;
 
-    this.createNewEditorHandler();
+    this._cellMap = new ObservableMap<EditorHandler>();
```

Comment:
This uses an `ObservableMap` instead of `Map` to follow what is used internally in the JupyerLab core notebook extension.

An `ObservableMap` is not strictly required in this case, but offers the possibility to listen on `changed` events.